ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -53,6 +58,17 @@ struct MSFButtonViewButtonStyle: ButtonStyle {
                         AnyView(RoundedRectangle(cornerRadius: tokens.borderRadius)
                                     .fill(Color(isDisabled ? tokens.disabledBackgroundColor :
                                                     (isPressed ? tokens.highlightedBackgroundColor : tokens.backgroundColor)))))
+        .modifyIf(tokens.style == .accentFloating || tokens.style == .subtleFloating, { view in
```

Comment:
this condition is referenced [here](https://github.com/microsoft/fluentui-apple/blob/76e2d98aef406d73c1a09d2cce434684c123fe3e/ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift#L18) too

You can add convenience method in enum `isFloatingStyle`

Reply:
I added the var, but I didn't think making it more accessible than internal was a good idea, so I didn't change the use in the demo controller. I don't think the users will generally have a need to access an internal var that will tell them what type of style they're trying to make.